backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java

```diff
@@ -188,7 +179,6 @@ public int processEvents(String playUuid,
                 }
             }
             lastEvent = event;
-            returnValue.setLastEventId(getLastEventId());
             lastEventId++;
         }
         return lastEvent.isEmpty() ? lastEventId : getLastEventId();
```

Comment:
Think that this patch still has the previous issue described in https://github.com/oVirt/ovirt-engine/pull/545.
IMHO the Michals patch looks best https://github.com/oVirt/ovirt-engine/pull/546 but this patch also contains nice refactor

Reply:
Yes, while I was going over the code trying to wrap my head around it I noticed some things that are unused. This one is not enough to fix the problem. Now rebased on top of Michal's changes.